### PR TITLE
Fix max login attempts msg

### PR DESCRIPTION
**Pull request by ValJed**
[1142](https://gitlab-dcadcx.michelin.net/pa/apostrophe-enhancements/-/issues/1142)

## Summary
Fixes missing plural label for login max attempts error message.
For normal login and requirement verification

## What are the specific steps to test this change?

* Link branch to A3 project
* Setup `lockoutMinutes` in login module config
```javascript
    '@apostrophecms/login': {
      options: {
        throttle: {
          lockoutMinutes: 2
        }
      }
    },
```
* Try to exceed max login attempt in initial login, the message should take into account the lockoutMinutes value
* Try to exceed max login attempt in totp login, the message should take into account the lockoutMinutes value

## What kind of change does this PR introduce?

- [X] Bug fix
- [ ] New feature
- [ ] Refactor
- [ ] Documentation
- [ ] Build-related changes
- [ ] Other

## Make sure the PR fulfills these requirements:

- [X] It includes a) the existing issue ID being resolved, b) a convincing reason for adding this feature, or c) a clear description of the bug it resolves
- [X] The changelog is updated
- [ ] Related documentation has been updated
- [ ] Related tests have been updated

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## unreleased
```
Comment (boutell) [inline CHANGELOG.md:3]: UNRELEASED

Comment (boutell) [inline CHANGELOG.md:3]: UNRELEASED

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
...
+
+### Fixes for
```
Comment (boutell) [inline CHANGELOG.md:5]: Just "Fixes"

**Review by boutell — CHANGES_REQUESTED** [review]

File: modules/@apostrophecms/login/index.js [hunk 3]
```diff
@@ -202,7 +202,13 @@ module.exports = {
             .checkLoginAttempts(user.username, loginNamespace);
 
           if (reached) {
-            throw self.apos.error('invalid', req.t('apostrophe:loginMaxAttemptsReached'));
+            const maxAttemptsKey = self.options.throttle.lockoutMinutes > 1
+              ? 'apostrophe:loginMaxAttemptsReached_plural'
```
[the remark this answers is not in the document]
Comment (ValJed) [reply modules/@apostrophecms/login/index.js]: Right, it's working well I just forgot the plural was handled automatically.

**Review by boutell — APPROVED** [review]

**Pull request merged**